src/index.spec.ts

```diff
@@ -4,87 +4,143 @@
 
 import { HttpClient } from '@eppo/js-client-sdk-common';
 import { POLL_INTERVAL_MS, POLL_JITTER_PCT } from '@eppo/js-client-sdk-common/dist/constants';
-import { IExperimentConfiguration } from '@eppo/js-client-sdk-common/dist/dto/experiment-configuration-dto';
-import { EppoValue } from '@eppo/js-client-sdk-common/dist/eppo_value';
-import * as md5 from 'md5';
+import { Flag, VariationType } from '@eppo/js-client-sdk-common/dist/interfaces';
 import * as td from 'testdouble';
 import mock from 'xhr-mock';
 
 import {
   IAssignmentTestCase,
-  ValueTestType,
+  SubjectTestCase,
   readAssignmentTestData,
-  readMockRacResponse,
+  readMockUFCResponse,
+  MOCK_UFC_RESPONSE_FILE,
+  OBFUSCATED_MOCK_UFC_RESPONSE_FILE,
 } from '../test/testHelpers';
 
 import { EppoLocalStorage } from './local-storage';
 import { LocalStorageAssignmentCache } from './local-storage-assignment-cache';
 
-import { IAssignmentLogger, IEppoClient, getInstance, init } from './index';
+import { EppoJSClient, IAssignmentLogger, IEppoClient, getInstance, init } from './index';
+
+const flagEndpoint = /flag_config\/v1\/config*/;
+
+function getTestAssignments(
```

Comment:
Food for possibly later thought:
We do this a lot, basically replicating the loading and then running test case logic in each SDK.

I wonder if instead, the SDK test repo has a program that runs and shells out to the command line.
Then each SDK just needs to implement a script that is takes a config, subject keys, attributes, and spits out the variations.

While we have common test _data_, this would ensure the _tests_ are all being run the same.